### PR TITLE
test(kms): add create kms integration tests

### PR DESCRIPTION
**Pull request by rjamet-ovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-15111
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by anooparveti — CHANGES_REQUESTED** [review]
Since it's addition of test-cases, can you change the type of commit to `test` instead?

**Review by pauldkn — APPROVED** [review]

File: packages/manager/apps/key-management-service/src/mocks/catalog/catalog.handler.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { Handler } from '../../../../../../../playwright-helpers';
```
Comment (anooparveti) [inline packages/manager/apps/key-management-service/src/mocks/catalog/catalog.handler.ts:1]: **question:** We will soon remove playwright from the repo.
Can you eliminate the dependency on it?

Comment (rjamet-ovh) [reply packages/manager/apps/key-management-service/src/mocks/catalog/catalog.handler.ts:1]: This type is cross product so it should be set on MRC or another shared dependency to be available for all apps.

Comment (tibs245) [reply packages/manager/apps/key-management-service/src/mocks/catalog/catalog.handler.ts:1]: For delete this dependencies we need to do have alternative of mock. I think can change the mock with Axios Mock : https://www.npmjs.com/package/axios-mock-adapter

And with this the test can be faster.

But to not create another new implementation. I agree with Romain. All units tests have this dependancies. 

It's possible to planned on control tower a example of implementation with the method of mock recommended ?

Maybe implementation on generator ?

File: packages/manager/apps/key-management-service/src/components/layout-helpers/Create/RegionSelector.tsx [hunk 2]
```diff
@@ -90,6 +90,7 @@ const RegionSelector = ({
           {orderCatalogOKMS && !isLoading && (
             <OsdsSelect
               inline
+              data-testid={'select-region'}
```
Comment (tibs245) [inline packages/manager/apps/key-management-service/src/components/layout-helpers/Create/RegionSelector.tsx:93]:
```suggestion
              data-testid='select-region'
```

**Review by tibs245 — APPROVED** [review]

File: packages/manager/apps/key-management-service/src/mocks/catalog/catalog.handler.ts (hunk 1 above)
Comment (rjamet-ovh) [reply packages/manager/apps/key-management-service/src/mocks/catalog/catalog.handler.ts:1]: we will clean our dependencies to the playwright folder on a specific development.

**Review by darsene — DISMISSED** [review]

**Review by pauldkn — DISMISSED** [review]

**Review by tibs245 — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by pauldkn — APPROVED** [review]

**Review by chipp972 — APPROVED** [review]

**Review by tibs245 — APPROVED** [review]

**Pull request merged**